### PR TITLE
Code review1 erich wagner

### PR DESCRIPTION
**Pull request by erswagner7**
About PictureThis:

A mobile application for deciding what movies to watch. Users can decide whether to pick a movie solo or with a group. As the user becomes interested in movies, the application will provide similar options that may catch the user's interest. After the user swipes the movies they show interest in, PictureThis will provide methods to access said movies.

This program was available on Github before the due date for the code review.

From what I've seen, the project is documented about 90% throughout, for the sections that have a significant amount of code. 

The project does compile and all tests pass on CircleCI, I am having trouble running it on the emulator (could be more on my end)

**Technical Suggestions**

1. I would recommend moving the DTO and DAO packages to be placed under com.example.picturethis and to make the package names lowercase to follow Kotlin naming conventions. https://kotlinlang.org/docs/coding-conventions.html#naming-rules
2. I would cut down on the number of unused lines of code (extra blank lines). This will reduce the technical debt the program has and will hopefully make it run faster and smoother. https://www.bmc.com/blogs/technical-debt-explained-the-complete-guide-to-understanding-and-dealing-with-technical-debt/
3. Make sure that if you have code you are using for an example, you remove it for the final product. In the tmdbApiDAO.kt file, there was a decent amount of the file made up of commented out code. https://www.bmc.com/blogs/technical-debt-explained-the-complete-guide-to-understanding-and-dealing-with-technical-debt/

**Three commits of mine:**

- https://github.com/zahercj/PictureThis/commit/5e516aa9d5ae8c6bbcf5968c03d7a89c5fdf08b2
- https://github.com/zahercj/PictureThis/commit/3e641105493a1490376f876dfe05914126bee730
- https://github.com/zahercj/PictureThis/commit/81c8fd1d11f0d777010808c58cff1b1c0d7c38ad

### CODE REVIEW

**Comment by erswagner7** [comment]
**Three Commits I made to my group's project**

1. https://github.com/erswagner7/PuzzlePic/commit/6623a18bd80ce41eb999d9db7b6bce7b7757d360
2. https://github.com/erswagner7/PuzzlePic/commit/1ad2f51217fff3631f688c1b25bc28f953b6f9cc
3. https://github.com/erswagner7/PuzzlePic/commit/ecc1c5caea5f3f223c1493ce5638ec1e971c5bec

**Comment by erswagner7** [comment]
At the moment, there aren't any specific technical concepts I've noticed that are implemented in the code at this point.
In the future though, I could see the following being used:

- Locationlistener: allow the user to see where in their area they could view the movies they're interested in
- Friends list: maybe a feature that accesses the user's contacts and allows them to invite others to the movie if they so wish
- Security: For those who would prefer keeping their movie preferences private from others

**What I would grade it as right now**

I would give it a low A, it needs more content to make a good first impression on someone looking at the project for the first time. I do see the potential in this app becoming really useful if implemented correctly.

File: app/src/main/java/com/example/picturethis/dao/movieDao.kt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.example.picturethis.dao
+
+class MovieDao(){
+
+
+}
```
Comment (discospiff) [inline app/src/main/java/com/example/picturethis/dao/movieDao.kt:6]: This appears to be an incomplete feature.  In a code review, focus on reducing technical debt of existing code.  Avoid adding new, incomplete functionality.
1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.

File: app/src/main/java/com/example/picturethis/dto/Movie.kt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package DTO
+package com.example.picturethis.dto
```
Comment (discospiff) [inline app/src/main/java/com/example/picturethis/dto/Movie.kt:1]: Good... though I'd go one step further, and choose a unique domain name.  "com.example" is not unique.

**Comment by discospiff** [comment]
From this review, I'd harvest the package name and case change.

Other changes are formatting and organizing imports, which can be done automatically by the IDE.  No need to merge a branch for what the IDE can do for you.

**Comment by zahercj** [comment]
Closing this PR and implementing the package name change manually. Other changes will be implemented manually.

**Pull request closed without merging**